lib/charms/tls_certificates_interface/v0/tls_certificates.py

```diff
@@ -449,6 +449,36 @@ def _parse_certificates_from_relation_data(relation_data: dict) -> List[Cert]:
                     )
         return certificates
 
+    def get_certificates_for_common_name(self, common_name: str = None, departed_id: int = None) -> List[Cert]:
```

Comment:
Why do we allow `common_name` to be none if the method is called `get_certificates_for_common_name`

Reply:
fixed in https://github.com/canonical/tls-certificates-interface/pull/4/commits/79e1543206e3b7a6e79f06203a7ff4adc3c8c076